apps/src/netsim/NetSimVisualization.js

```diff
@@ -143,10 +143,11 @@ NetSimVisualization.prototype.tick = function (clock) {
 
 /**
  * Render: Let all vizentities "redraw" (or in our case, touch the DOM)
+ * @param {RunLoop.Clock} clock
  */
-NetSimVisualization.prototype.render = function () {
+NetSimVisualization.prototype.render = function (clock) {
```

Comment:
does this PR include a call to render with the clock parameter?  or is that maybe already in the code?